src/App.js

```diff
@@ -1,316 +1,21 @@
 import "./App.css";
-import heroHeader from "./assets/hero-header.png"
-import spartanLogo from "./assets/spartan-logo.png"
-import nikeLogo from "./assets/nike-logo.png"
-import mrfLogo from "./assets/mrf-logo.png"
-import kookaburraLogo from "./assets/kookaburra-logo.jpg"
-import newbalanceLogo from "./assets/newbalance-logo.png"
-import cricketBall from "./assets/cricket-ball.jpg"
-import jersey from "./assets/jersey.png"
-import pumaLogo from "./assets/Logo+Puma.jpg"
-import kookaburraBall from "./assets/kookabura-ball.png"
-import miExclusive from "./assets/mi-exclusive.png"
-import mrfBat from "./assets/mrf-bat.png"
-import nbBat from "./assets/nb-bat.jpg"
-import nikeFootball from "./assets/nike-football.png"
-import nikeShoes from "./assets/nike-shoes.png"
-import spartanBat from "./assets/spartan-bat.jpg"
-
+import spartanLogo from "./assets/spartan-logo.png";
+import nikeLogo from "./assets/nike-logo.png";
+import mrfLogo from "./assets/mrf-logo.png";
+import kookaburraLogo from "./assets/kookaburra-logo.jpg";
+import newbalanceLogo from "./assets/newbalance-logo.png";
+import cricketBall from "./assets/cricket-ball.jpg";
+import pumaLogo from "./assets/Logo+Puma.jpg";
+import miExclusive from "./assets/mi-exclusive.png";
+import nbBat from "./assets/nb-bat.jpg";
+import spartanBat from "./assets/spartan-bat.jpg";
+import { LandingPage } from "./pages/LandingPage";
+import nikeShoes from "./assets/nike-shoes.png";
```

Comment:
The imports are not being used here, it's better to move them in the respective components.